packages/server/src/service/trpc.ts

```diff
@@ -119,31 +115,9 @@ export class VbmRpc implements ImplRpc<IVbmRpc, Request> {
       method,
     })
 
-    return data(id, async (): Promise<void> => {
-      const letter = new Letter(info, method, id)
-      const pdfBuffer = await toPdfBuffer(letter.render('html'), await letter.form)
-
-      // Send email (perhaps only to voter)
-      const mgResponse = await sendSignupEmail(
-        letter,
-        info.email,
-        method.emails,
-        { pdfBuffer, id },
-      )
-
-      // Upload PDF
-      const file = storageFileFromId(id)
-      await file.upload(pdfBuffer)
-
-      // Send faxes
-      let twilioResponses: TwilioResponse[] = []
-      if (method.faxes.length > 0) {
-        const uri = await file.getSignedUrl(24 * 60 * 60 * 1000)
-        const resposnes = await sendFaxes(uri, method.faxes)
-        twilioResponses = resposnes.map(({url, sid, status}) => ({url, sid, status}))
-      }
-
-      await firestoreService.updateRegistration(id, mgResponse, twilioResponses)
-    })
+    const success = await sendAndStoreSignup(info, method, id)
```

Comment:
Hi -- no: please return the function instead of this.  Ths only slows things down and doesn't give the user information they can respond to.  Instead, I would suggest putting this function in a retry to get rid of transient errors.  Can you add the following comment below?

The email and fax APIs do not immediately return success (just pending) so there is no additional information to be gained.  However, it can take them some time to do this.  Once we have the firebase database record, we return the result to the user and process the email and fax asynchronously in the background.